content/concepts/products.md

```diff
@@ -509,6 +509,47 @@ Whenever the attribute's type is `pim_catalog_asset_collection`, the `data` fiel
 }
 ```
 
+#### Table attribute
+::: availability version=SaaS editions=EE,GE
+:::
+
+Whenever the attribute's type is `pim_catalog_table`, the `data` field should contain an array of rows, where each row is a key-value object, the key being the `column` code, and the value being the cell value.
+
+##### Example
+```json
+{
+  "values": {
+    "nutrition": [
+      {
+        "data": [
+          {
+              "average_nutritional_value": "energy",
+              "per_100g": "248",
```

Comment:
With this example, it seems we have to put all numbers in string. But if the column is a number column, we should put the number not as a string

Reply:
j'ai pas tout compris nini...